src/ccs/confluent.sh

```diff
@@ -993,6 +993,17 @@ list_command() {
     fi
 }
 
+# CLI-84: If using Mac, version must be >= 10.13
+validate_os_version() {
+    local OS=$(uname -s)
+    if [[ "${OS}" = "Darwin" ]]; then
+        local mac_version=$(sw_vers -productVersion)
+        if [[ $(awk 'BEGIN {print ("'${mac_version}'" < "'10.16'")}') )) ]]; then
```

Comment:
Shouldn't 10.16 be 10.13?  (I assume you used 10.16 just for testing ;) )

Reply:
Yep, was just testing ; )